plugins/chat/db/migrate/20250903091723_create_chat_message_search_data.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreateChatMessageSearchData < ActiveRecord::Migration[7.1]
+  def change
+    create_table :chat_message_search_data, id: false do |t|
```

Comment:
Keep in mind if we deploy anywhere for testing we need to update this migration to work a little differently for the existing table + indices